raiden_libs/utils/signing.py

```diff
@@ -101,3 +103,13 @@ def address_from_signature(sig: bytes, msg: bytes):
 
 def eth_verify(sig: bytes, msg: str) -> str:
     return address_from_signature(sig, keccak256(msg))
+
+
+def sign_transaction(tx: Transaction, privkey: str, network_id: int):
+    # Implementing EIP 155.
```

Comment:
Make this a docstring. Also a test would be nice.

Reply:
test is already included... if signature doesn't work, local signing won't work :)